examples/config/config

```diff
@@ -122,6 +122,26 @@ set completion_style  foreground="#eee8d5"
 set hint_style        weight="bold"
 
 set mode_section      <span background="#6c71c4" foreground="#eee8d5">[\@[\@mode_indicator]\@]</span>
+
+# As these indicators slow down uzbl, they're disabled by default
+set view_mode_indicator   \@- if ('\@page_view_mode' == 'web')   '<span foreground="#eee8d5" background="#586e75">'; else '<span foreground="#586e75" background="#eee8d5">'; -\@VM</span>
+# TODO find out why this fails
+#set css_indicator         \@- if (document.getElementsByTagName("link").length === 0)  '<span foreground="#eee8d5">'; else '<span foreground="#eee8d5" background="#2aa198">'; -\@CS</span>
```

Comment:
This is likely failing because @- runs in a clean js environment which is not the same as where the DOM of the page lives